junit-platform-commons/src/main/java/org/junit/platform/commons/util/CollectionUtils.java

```diff
@@ -138,8 +140,12 @@ public static boolean isConvertibleToStream(Class<?> type) {
 				|| LongStream.class.isAssignableFrom(type)//
 				|| Iterable.class.isAssignableFrom(type)//
 				|| Iterator.class.isAssignableFrom(type)//
+				|| Spliterator.class.isAssignableFrom(type)//
```

Comment:
Why did you add support for `Spliterator`? That's not needed to support Kotlin sequences, is it?

Reply:
That's true. Maybe scope creep got to me. I will adapt it to narrow the scope and implement it as follows: Add convention-based conversion to Stream that looks for methods called iterator that return Iterator.